dotCMS/src/main/java/com/liferay/portlet/PortletUtil.java

```diff
@@ -0,0 +1,94 @@
+package com.liferay.portlet;
+
+import com.dotcms.repackage.org.directwebremoting.WebContext;
+import com.dotcms.repackage.org.directwebremoting.WebContextFactory;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.business.web.WebAPILocator;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.portlets.user.ajax.UserAjax;
+import com.dotmarketing.util.SecurityLogger;
+import com.liferay.portal.PortalException;
+import com.liferay.portal.SystemException;
+import com.liferay.portal.model.User;
+import javax.servlet.http.HttpServletRequest;
+
+/**
+ * @author Jonathan Gamba 11/29/17
+ */
+public class PortletUtil {
+
+    /**
+     * Validates if the logged in user has the required permissions to access the users portlet
+     *
+     * @param loggedInUser User to validate
+     */
+    public static void validateUsersPortletPermissions(User loggedInUser)
+            throws DotSecurityException, DotDataException, SystemException, PortalException {
+
+        validatePortletPermissions("users", loggedInUser);
+    }
+
+    /**
+     * Validates if the logged in user has the required permissions to access the roles portlet
+     *
+     * @param loggedInUser User to validate
+     */
+    public static void validateRolesPortletPermissions(User loggedInUser)
+            throws DotSecurityException, DotDataException, SystemException, PortalException {
+
+        validatePortletPermissions("roles", loggedInUser);
+    }
+
+    /**
+     * Validates if the logged in user has the required permissions to access the given portlet
+     *
+     * @param portletId Portlet to validate
+     * @param loggedInUser User to validate
+     */
+    public static void validatePortletPermissions(String portletId, User loggedInUser)
+            throws DotSecurityException, DotDataException, SystemException, PortalException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.liferay.portal.SystemException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.liferay.portal.PortalException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)